modules/nw-ovn-kubernetes-rollback-live.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-kubernetes-rollback-live_{context}"]
+= Using the live migration method to roll back to the OpenShift SDN network plugin
+
+As a cluster administrator, you can rollback to the OpenShift SDN Container Network Interface (CNI) network plugin by using the live migration method. During the migration with this method, nodes are automatically rebooted and service to the cluster is not interrupted.
+
+[IMPORTANT]
+====
+You must wait until the migration process from OpenShift SDN to OVN-Kubernetes network plugin is successful before initiating a rollback.
+====
+
+If a rollback to OpenShift SDN is required, the following table describes the process.
+
+.Performing a rollback to OpenShift SDN
+[cols="1,1a",options="header"]
+|===
+
+|User-initiated steps|Migration activity
+ifdef::openshift-rosa,openshift-dedicated[]
+| Add the `unsupported-red-hat-internal-testing` annotation to the cluster-level network configuration. 
+| The Cluster Network Operator (CNO) acknowledges the unsupported testing environment.
+endif::[]
+
+| Patch the cluster-level networking configuration by changing the `networkType` from `OVNKubernetes` to `OpenShiftSDN`.
+| 
+Cluster Network Operator (CNO)::
++
+--
+* Sets migration related fields in the `network.operator` custom resource (CR) and waits for routable MTUs to be applied to all nodes by MCO.
+* Patches the `network.operator` CR to set the migration mode to `Live` for OpenShiftSDN and deploys the OpenShiftSDN network plugin in migration mode.
```

Comment:
Previous instances are:
```suggestion
* Patches the `network.operator` CR to set the migration mode to `Live` for OpenShift SDN and deploys the OpenShift SDN network plugin in migration mode.
```
As an aside, should "OpenShift SDN" have an attribute to make it easier to keep up with and avoid having "OpenShift" in plain text? If you think it's worthwhile, maybe a thing to do later :)

Reply:
It's deprecated now and is being removed next release, otherwise I would :)